src/components/pageTemplates/NodeBasicPage.tsx

```diff
@@ -17,40 +21,57 @@ function NodeBasicPage({ node, sidebar, ...props }: NodeBasicPageProps): JSX.Ele
     field_hide_sidebar,
     field_lower_content,
     langcode,
-  } = node
+    field_content_language,
+  } = node;
+
+  const contentLang =
+    field_content_language?.resourceIdObjMeta.drupal_internal__target_id;
```

Comment:
This field has been removed because we ended up not needing the funtionality after all, so it should be removed from there too.